setup/setup.nsi

```diff
@@ -696,6 +696,11 @@ Function .onInit
 		ReadRegDword $0 HKLM "${REGPATH_POSREADY}" "Installed"
 		${If} $0 == 1
 			!insertmacro RemoveSection ${WES09}
+		${Else}
+			ReadRegDword $0 HKLM "${REGPATH_WES}" "Installed"
+			${If} $0 == 1
+				!insertmacro RemoveSection ${WES09}
+			${EndIf}
```

Comment:
We don’t want to do this, because POSReady got updates for a bit longer than WES. Adding the POSReady key on WES is safe to do, extends the final updates from Jan 8, 2019 to Apr 9, 2019.

Reply:
This has been removed, I incorrectly assumed Standard had the same support length as POSReady.